content/source/docs/enterprise/sentinel/import/tfplan.html.md

```diff
@@ -8,258 +8,505 @@ description: |-
+```python
+import "tfplan"
 
-This is the old value. If it wasn't set before, the returned value
-will be an empty string. It is not possible to distinguish between
-an old value of an empty string and the value not being set before.
+main = rule { tfplan.resources.null_resource.foo[0].applied.triggers.foo is "bar" }
+```
 
-This value can be used to enforce policy on the types of changes that
-can occur in certain attribute values. For example, there may be a policy
-that tag values can only be appended to, and not replaced.
+### Value: `diff`
 
-### attribute.new
+* **Value Type:** A map of [diff namespaces](#namespace-resource-diff).
 
-This is the new value that the attribute is going to, as a string.
-This may be the empty string if the value is computed. You can check if
-the value is computed using the `computed` field.
+The `diff` value within the [resource
+namespace](#namespace-resources-data-sources) contains the diff for a particular
+resource. Each key within the map links to a [diff
+namespace](#namespace-resource-diff) for that particular key.
 
-### attribute.computed
+Note that unlike the [`applied`](#value-applied) value, this map is not complex;
+the map is only 1 level deep with each key possibly representing a diff for a
+particular complex value within the resource.
 
-This is a boolean value that is true if this attribute's new value is
-a computed value. A computed value is a value that is not known until
-apply time. This can be because this is a dynamic value from this
-resource itself such as the IP address of a new compute instance, or
-because it is a parameter that is being populated from a dynamic
-value from another resource.
+See the diff namespace for specific examples.
 
-If a value is computed, it cannot be known until after apply. If policy
-must be enforced on the value of this field, it must be done post-apply
-or by disallowing computed values.
+## Namespace: Resource Diff
 
-### attribute.removed
+The **diff namespace** is a namespace that represents the diff for a specific
+attribute within a resource. For particulars on reading a particular attribute,
+see the [`diff`](#diff) value in the [resource
+namespace](#namespace-resources-data-sources).
 
-This is a boolean value that is true if this field is being removed.
-This can be set if the entire configuration is deleted, or if the element
-of a map or list is being removed. This is set for example when a tag is
-removed from an AWS resource.
+### Value: `computed`
 
-When a value is removed, its `new` value will be an empty string.
+* **Value Type:** Boolean.
 
-## Type: module
+The `computed` value within the [diff namespace](#namespace-resource-diff) is
+`true` if a value is currently unknown in the diff, but is changing. This
+happens when a value depends on a value belonging to a resource that either does
+not exist yet or is changing state in a way that the new value will not be known
+until the apply for that resource completes.
```

Comment:
I'd like to keep the syntax/grammar of the first sentence, so I've blended this in to it. New paragraph:

> The `computed` value within the diff namespace is `true` if the resource key in question depends on another value that isn't yet known. Typically, that means the value it depends on belongs to a resource that either doesn't exist yet, or is changing state in such a way as to affect the dependent value so that it can't be known until the apply is complete.